SU-SWS/stanford_paragraph_types
# Hsdo 939 section header

# READY FOR REVIEW

# Summary
- Relies on https://github.com/SU-SWS/stanford_paragraph_types/pull/40
- New simple paragraph type that is just a h2 header page separator.

# Needed By (Date)
- End of sprint

# Urgency
- low

# Steps to Test

1. Checkout branch
2. `drush en stanford_paragraph_types_section_header -y`
3. Create a stanford_page and use the `Section Header` paragraph type with some color.
4. output should just be an H2 tag with some background color.

# See Also
- https://github.com/SU-SWS/stanford_paragraph_types/pull/40
- [linky_clicky](https://github.com/SU-SWS/linky_clicky/pull/171)
